server/cards/cards.go

```diff
@@ -0,0 +1,73 @@
+package cards
```

Comment:
This file was pretty much left unchanged.  It is just a move from the actions package.